qa/rpc-tests/dip4-coinbasemerkleroots.py

```diff
@@ -81,6 +81,79 @@ def run_test(self):
         # When comparing genesis and best block, we shouldn't see the previously added and then deleted MN
         mnList = self.test_getmnlistdiff(null_hash, self.nodes[0].getbestblockhash(), {}, [], expectedUpdated2)
 
+        #############################
+        # Now start testing quorum commitment merkle roots
+
+        height = self.nodes[0].getblockcount()
+        # Test DIP8 activation once with a pre-existing quorum and once without (we don't know in which order it will activate on mainnet)
+        self.test_dip8_quorum_merkle_root_activation(True)
+        for n in self.nodes:
+            n.invalidateblock(n.getblockhash(height + 1))
```

Comment:
```suggestion
            n.invalidateblock(oldhash)
```
otherwise we kind of relying on the internal logic of `test_dip8_quorum_merkle_root_activation` (we expect it to mine at least one block), which is not smth this line is about, we just want to reset the chain here.